fix_formatting.py

```
@@ -0,0 +1,639 @@
+#!/usr/bin/env python3
+"""
+Script to fix formatting issues in Python files.
+"""
+
+import os
+import sys
+import subprocess
+from pathlib import Path
+
+
+def fix_file(file_path):
```

Ruff (0.11.9)

Missing return type annotation for public function `fix_file`

Add return type annotation: `None`

(ANN201)